themes/src/main/resources/theme/keycloak-preview/account/resources/app/content/my-resources-page/PermissionRequest.tsx

```diff
@@ -127,7 +127,9 @@ export class PermissionRequest extends React.Component<PermissionRequestProps, P
                                                 {shareRequest.scopes.map((scope, j) => <Chip key={j} isReadOnly>{scope}</Chip>)}
                                             </DataListCell>,
                                             <DataListCell key={`actions${i}`}>
-                                                <Button onClick={() => this.handleApprove(shareRequest.username, shareRequest.scopes)}>
+                                                <Button
+                                                    style={{marginRight: '10px'}}
```

Comment:
Can we find a PatternFly way to deal with this?  Granted, I haven't run this to see what it is about, but this sort of thing is always a red flag.

Reply:
changed to use Stack